lib/internal/modules/cjs/helpers.js

```diff
@@ -110,11 +110,40 @@ function stripBOM(content) {
 }
 
 const builtinLibs = [
-  'assert', 'async_hooks', 'buffer', 'child_process', 'cluster', 'crypto',
```

Comment:
cosmetic?

Reply:
Yes and no. Let's say convenience. Changes to this list end up being overly disruptive and difficult to review. Separating into separate lines makes the list more manageable overall. That said, given that we're not using the top level module anymore and we're using 'net' instead, we revert this change.